omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -27,21 +27,20 @@
 public class MarkPatientDeadPageController {
     protected final Log log = LogFactory.getLog(this.getClass());
 
-    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") String patientId) {
+    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") Patient patient) {
```

Comment:
Didn't we agree that the above change is gonna be in a separate pull request?

Reply:
@dkayiwa  this is the separate PR. with only one issue. I don't understand how micro you want this to get.

Reply:
Oh i see!
I now get you. 😊